src/plugins/saved_objects/public/saved_object_finder.tsx

```diff
@@ -22,10 +22,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import {
```

Comment:
I would put these in a components folder

Reply:
I moved it to the`finder` folder.